WooCommerce/Classes/Analytics/WooAnalyticsEvent.swift

```diff
@@ -75,6 +75,12 @@ extension WooAnalyticsEvent {
         case orderCreation = "order_creation"
         /// Shown in beta feature banner for coupon management.
         case couponManagement = "coupon_management"
+        /// Shown in IPP banner for eligible merchants with no IPP transactions.
+        case IPP_COD
+        /// Shown in IPP banner for eligible merchants with a few IPP transactions.
+        case IPP_firstTransaction
+        /// Shown in IPP banner for eligible merchants with a significant number of IPP transactions.
+        case IPP_powerUsers
```

Comment:
```suggestion
        case inPersonPaymentsCashOnDeliveryBanner
        /// Shown in IPP banner for eligible merchants with a few IPP transactions.
        case inPersonPaymentsFirstTransactionBanner
        /// Shown in IPP banner for eligible merchants with a significant number of IPP transactions.
        case inPersonPaymentsPowerUsersBanner
```
Abbreviations, snake case, and capitalisation of IPP look a bit out of place here. Is this for consistency with Android? If so it could be handled with a rawValue string or something, perhaps...

Reply:
Definitely look better now, changed here d6a45ed4ea63d01e8953d6c1a8fd684eaced4a4f